Client/src/Components/product-overview/StyleSelector.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useContext } from "react";
+import DataContext from "../context";
+import css from "./styles.css";
+
+function StyleSelector() {
+  const data = useContext(DataContext);
+  const { styleIndex, styles } = data;
+  const [currentSelect, setCurrentSelect] = useState(0);
+  const sizes = [];
+  const quantitySelector = (max) => {
+    // input: number (quantity from sku)
+    // output: array of numbers 1-max or 15
+    // purpose: creates the values for the quantity selector
+    let newMax = max;
+    if (newMax > 15) {
+      newMax = 15;
+    }
+    return Array.from(Array(newMax + 1).keys()).slice(1);
+  };
+  const getSizes = () => {
```

Comment:
consider putting quanitySelector and getSizes in a separate file for helper functions. It would keep the StyleSelector cleaner